test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
```

Comment:
I think you should consider calling this `test/integration_test_helper.rb`. The test suffix I feel is often associated with an actual test to run, whereas this is more of a helper. Perhaps we should consider even putting this in the `helpers/` directory?